index.html

```diff
@@ -225,6 +225,46 @@ <h4>Skills</h4>
         </div>
       </div>
     </section>
+    <section>
+      <div class="contact-container">
+        <h1 class="contact-heading">Contact me</h1>
+        <p class="contact-paragraph">
+          If you have an application you are interested in developing, a feature
+          that you need built or a project that needs coding, I'd love to help
+          with it.
+        </p>
+        <form class="contact-form" action="https://formspree.io/f/mjvdqdaj method="POST">
+          <input
+            class="contact-input"
+            required
+            maxlength="30"
+            name="name"
+            type="text"
+            placeholder="Please enter your name (max 30 characters)"
+          />
+          <input
+            class="contact-input"
+            required
+            name="email"
+            type="email"
+            pattern="[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}"
+            placeholder="Your Email address"
+          />
+          <textarea
+          class="contact-input"
+          required
+          maxlength="500"
+          name="message"
+          cols="30"
+          rows="9"
+          placeholder="Write your message here (up to 500 characters)"
+        ></textarea>
+        <button class="contact-button" type="submit">
+          Get In Touch
+        </button>
```

Comment:
- [ ] Well, great job on implementing default and hover styles for your buttons! 👍 However, upon reviewing your code, I noticed that the styles applied to the buttons are not consistent with the design. Additionally, the design specifies that the buttons should also have active styles. 🙏 Please consider updating your button styles to include the active state, following the design closely. This will provide a complete set of styles for the buttons, ensuring a consistent and visually appealing user experience. Keep up the good work! 💪
![image](https://github.com/the-faizmohammad/my-portfolio/assets/55840999/e78f09db-78c7-469d-843a-f52d72cfe7a1)